### PR TITLE
Bump up inabox error log severity level, so that it will show up in StackDriver

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: tools/errortracker/errortracker.go [hunk 1]
```diff
@@ -129,6 +129,9 @@ func handle(w http.ResponseWriter, r *http.Request) {
 	runtime := r.URL.Query().Get("rt")
 	if runtime != "" {
 		errorType += "-" + runtime;
+		if runtime == "inabox" {
+			severity = "ERROR"
```
Comment (cramforce) [inline tools/errortracker/errortracker.go]: Also need to set `level`

Comment (lannka) [inline tools/errortracker/errortracker.go]: done

**Comment by lannka** [comment]
pls help deploy :-)

**Comment by cramforce** [comment]
Done

**Pull request merged**